b2sdk/raw_simulator.py

```diff
@@ -1079,8 +1078,11 @@ def upload_part(
         computed_sha1 = hex_sha1_of_bytes(part_data)
         if sha1_sum != computed_sha1:
             raise PartSha1Mismatch(file_id)
+
+        file_sim = self.file_id_to_file[file_id]
```

Comment:
unrelated change, but it improves the code locality a bit - `file_sim` isnt used anywhere between its definition and the next line